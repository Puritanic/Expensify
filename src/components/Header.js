import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = props => (
    <header>
        <h2>Expensify</h2>
        <NavLink to="/" exact activeClassName="is-active">
            Dashboard
        </NavLink>
        <NavLink to="/create" activeClassName="is-active">
            Add Expense
        </NavLink>
        <NavLink to="/edit" activeClassName="is-active">
            Edit Expense
        </NavLink>
        <NavLink to="/help" activeClassName="is-active">
            Need Help?
        </NavLink>
    </header>
);

export default Header;
