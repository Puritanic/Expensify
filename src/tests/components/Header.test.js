import React from 'react';
// import { shallow } from 'enzyme';
import Header from '../../components/Header';
/* eslint-disable no-undef */
it('should render Header correctly', () => {
    const wrapper = shallow(<Header />);
    expect(toJson(wrapper)).toMatchSnapshot();
    // expect(wrapper.find('h2').length).toBe(1);
    // expect(wrapper.find('h2').text()).toBe('Expensify');
});
/* eslint-enable no-undef */
