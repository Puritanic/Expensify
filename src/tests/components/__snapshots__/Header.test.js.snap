// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header
  className="header"
>
  <Link
    className="header__title"
    exact="true"
    replace={false}
    to="/dashboard"
  >
    <h2>
      Expensify
    </h2>
  </Link>
  <button
    className="button"
    onClick={[Function]}
  >
    Logout
  </button>
</header>
`;
